.github/workflows/master-test.yaml

```diff
@@ -18,3 +18,21 @@ jobs:
       - name: Master buld tests
         run: |
           make test   
+  functional:
+    name: Run functional tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Start up dependencies
+        run: docker-compose up -d
+      - name: Wait for dependencies to start up
+        uses: jakejarvis/wait-action@master
+        with:
+          time: '30s'
+      - name: Test
+        env:
+          AWS_REGION: us-west-2
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
Which key/access id is this?

Reply:
The one Fritz made specifically for this